rct229/rules/section5/section5rule26.py

```diff
@@ -0,0 +1,274 @@
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=10.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=20.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=30.1,
+                    )["solar_heat_gain_coefficient"]
+                )
+                and (
+                    table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_NON_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=0.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_NON_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=10.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_NON_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=20.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_NON_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=30.1,
+                    )["solar_heat_gain_coefficient"]
+                )
+                and (
+                    table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=0.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_NON_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=0.1,
+                    )["solar_heat_gain_coefficient"]
+                )
+            )
+            # get standard code data
+            target_shgc_mix = (
+                table_G34_lookup(
+                    climate_zone,
+                    SCC.EXTERIOR_RESIDENTIAL,
+                    "VERTICAL GLAZING",
+                    wwr=bldg_scc_wwr_ratio[SCC.EXTERIOR_MIXED],
+                )["solar_heat_gain_coefficient"]
+                if bldg_scc_wwr_ratio[SCC.EXTERIOR_MIXED] > 0
+                else 0.0
```

Comment:
I guess the `0.0` value is saying that a wall with no fenestration has a solar heat gain coefficient of `0.0`.  That makes sense.

Should this logic be added to the RDS?

Reply:
I actually think None may work better in this case to avoid confusions in the report.